docker/Dockerfile-builder

```diff
@@ -11,3 +11,7 @@ WORKDIR /gem5
 RUN scons -j`nproc` build/X86/gem5.fast CPU_MODELS=AtomicSimpleCPU,TimingSimpleCPU,O3CPU,MinorCPU
 RUN scons -j`nproc` build/ARM/gem5.fast CPU_MODELS=AtomicSimpleCPU,TimingSimpleCPU,O3CPU,MinorCPU
 RUN scons -j`nproc` build/RISCV/gem5.fast CPU_MODELS=AtomicSimpleCPU,TimingSimpleCPU,O3CPU,MinorCPU
+
+# Install Python library dependencies
+RUN apt -y install python3-pip
+RUN pip3 install seaborn
```

Comment:
I'm pretty sure this is in the wrong file. You want to put these in the docker/Dockerfile if you want them to be in the codespace environment.. Pip is already installed.

Reply:
is it `pip` or `pip3`?

Reply:
Run `which pip` in the current codespace container...